utils/src/blockuntilzero.rs

```diff
@@ -56,7 +56,6 @@ where
         self.value.load(Ordering::Acquire)
     }
 
-    #[must_use = "CountTracker is useless without holding its object"]
```

Comment:
This isn't an error: the `CountTracker` type already has the `must_use` attribute, so this is redundant.